continuous_integration/environment-mindeps-optional.yaml

```diff
@@ -0,0 +1,59 @@
+name: test-environment
+channels:
+  - conda-forge
+dependencies:
+  # required dependencies
+  - packaging=20.0
+  - python=3.8
+  - pyyaml=5.3.1
+  - click=8.0  # min version required by distributed
+  - cloudpickle=1.5.0  # min version required by distributed
```

Comment:
After internal discussion with @jrbourbeau, we agreed to bump the versions of shared Dask/Distributed dependencies so that they share the same min version.

Is there a preference between doing that in this PR, versus separately?

Reply:
Either is fine by me, whichever you prefer

Reply:
Cool, will go ahead and do it here then